addOns/oast/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added 
 - API support.
 - Raise alerts for OAST interactions that happened in other sessions.
+- Options to trim the database.
```

Comment:
Be more specific that this is OAST or Permanent?

Reply:
Well, its in the OAST add-on, but yeah, its trimming the OWASP perm db records

Reply:
Updated